src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -178,7 +178,7 @@ std::vector<std::string> disabledTestPatterns() {
         // Old API cannot deallocate tensor
         R"(.*InferRequestIOBBlobTest.*canProcessDeallocatedOutputBlobAfterGetAndSetBlob.*)",
         // Plugin version was changed to ov::Version
-        R"(.*VersionTest.*pluginCurrentVersionIsCorrect.*)",
+        R"(.*VersionTest.pluginCurrentVersionIsCorrect.*)",
```

Comment:
Can we remove this line to enable this tests?

Reply:
No. This skip is for 1.0 test, 2.0 tests had worked.